frontend/tests/features/cellCards/cellCards.test.ts

```diff
@@ -74,11 +86,17 @@ describe("Cell Guide", () => {
       const firstOption = (await page.getByRole("option").elementHandles())[0];
       const firstOptionText = await firstOption?.textContent();
       expect(firstOptionText).toBe("neuron");
-      // click on first element in dropdown
-      await firstOption?.click();
-      // check that the url has changed to the correct CellGuide card
-      await page.waitForURL(`${TEST_URL}${ROUTES.CELL_GUIDE}/CL_0000540`); // Neuron
+
+      await Promise.all([
```

Comment:
wrap click and waitForURL in one `Promise.all()` to prevent race condition stalling the test caused by page navigation happens before waitForURL is invoked - here and throughout